app/controllers/brands_controller.rb

```diff
@@ -1,2 +1,8 @@
 class BrandsController < ApplicationController
+
+  def explore
+    @brands = Brand.all
+    @newsarticles = NewsArticle.all
```

Comment:
Add a variable and calculation for the brand's average global rating here? using the all the ratings from the brands table @flaminiavg